pandas/io/formats/terminal.py

```diff
@@ -94,21 +94,26 @@ def _get_terminal_size_tput():
     # get terminal width
     # src: http://stackoverflow.com/questions/263890/how-do-i-find-the-width
     # -height-of-a-terminal-window
+    output_cols = None
+    output_rows = None
     try:
         import subprocess
```

Comment:
can you move the import to the top of the file

Reply:
Sure! How do I add this whatsnew note, by the way?

As for the tests, I can try, but having never done that before, I can't guarantee it'll go very well

Reply:
edit doc/source/whatsnew/v0.24.2.rst